src/components/TodoComponents/Todo.css

```diff
@@ -0,0 +1,74 @@
+h1 {
```

Comment:
i would really love to see a style for completed todos so that they're visually different